pygments/lexers/_stan_builtins.py

```diff
@@ -253,102 +330,163 @@
     'modified_bessel_first_kind',
     'modified_bessel_second_kind',
     'multi_gp_cholesky_lpdf',
+    'multi_gp_cholesky_lupdf',
     'multi_gp_lpdf',
+    'multi_gp_lupdf',
     'multi_normal_cholesky_lpdf',
+    'multi_normal_cholesky_lupdf',
     'multi_normal_cholesky_rng',
     'multi_normal_lpdf',
+    'multi_normal_lupdf',
     'multi_normal_prec_lpdf',
+    'multi_normal_prec_lupdf',
     'multi_normal_rng',
     'multi_student_t_lpdf',
+    'multi_student_t_lupdf',
     'multi_student_t_rng',
+    'multinomial_logit_lpmf',
+    'multinomial_logit_lupmf',
+    'multinomial_logit_rng',
     'multinomial_lpmf',
+    'multinomial_lupmf',
     'multinomial_rng',
     'multiply_log',
     'multiply_lower_tri_self_transpose',
     'neg_binomial_2_cdf',
     'neg_binomial_2_lccdf',
     'neg_binomial_2_lcdf',
+    'neg_binomial_2_log_glm_lpmf',
+    'neg_binomial_2_log_glm_lupmf',
     'neg_binomial_2_log_lpmf',
+    'neg_binomial_2_log_lupmf',
     'neg_binomial_2_log_rng',
     'neg_binomial_2_lpmf',
+    'neg_binomial_2_lupmf',
     'neg_binomial_2_rng',
     'neg_binomial_cdf',
     'neg_binomial_lccdf',
     'neg_binomial_lcdf',
     'neg_binomial_lpmf',
+    'neg_binomial_lupmf',
     'neg_binomial_rng',
     'negative_infinity',
+    'norm',
     'normal_cdf',
+    'normal_id_glm_lpdf',
+    'normal_id_glm_lupdf',
     'normal_lccdf',
     'normal_lcdf',
     'normal_lpdf',
+    'normal_lupdf',
     'normal_rng',
     'not_a_number',
     'num_elements',
+    'ode_adams',
+    'ode_adams_tol',
+    'ode_adjoint_tol_ctl',
+    'ode_bdf',
+    'ode_bdf_tol',
+    'ode_ckrk',
+    'ode_ckrk_tol',
+    'ode_rk45',
+    'ode_rk45_tol',
+    'one_hot_array',
+    'one_hot_int_array',
+    'one_hot_row_vector',
+    'one_hot_vector',
+    'ones_array',
+    'ones_int_array',
+    'ones_row_vector',
+    'ones_vector',
+    'ordered_logistic_glm_lpmf',
+    'ordered_logistic_glm_lupmf',
     'ordered_logistic_lpmf',
+    'ordered_logistic_lupmf',
     'ordered_logistic_rng',
+    'ordered_probit_lpmf',
+    'ordered_probit_lupmf',
+    'ordered_probit_rng',
     'owens_t',
     'pareto_cdf',
     'pareto_lccdf',
     'pareto_lcdf',
     'pareto_lpdf',
+    'pareto_lupdf',
     'pareto_rng',
     'pareto_type_2_cdf',
     'pareto_type_2_lccdf',
     'pareto_type_2_lcdf',
     'pareto_type_2_lpdf',
+    'pareto_type_2_lupdf',
     'pareto_type_2_rng',
-    'Phi',
-    'Phi_approx',
```

Comment:
These weren’t truly removed, the script I used to print the function names just put them at the start since they begin with capital letters. I’ll move them back and also add back `print` and `reject`